src/Network/Wai/Log/Options.hs

```diff
@@ -2,26 +2,29 @@ module Network.Wai.Log.Options (
 -- * Options & Timing
   Options(..)
 , ResponseTime(..)
+, logSendingResponse
```

Comment:
We should move `logSendingResponse` directly into `Log.Internal` as it is useless for it to be exposed as part of the library since it cannot be used with `Options` anymore (on purpose—that's our design choice).

Reply:
But it's nice that all functions that create `[Pair]` values for logging are here, next one to each other. It is not re-exported in `Log.hs`, and I've moved it to `other-modules` so that it cannot be used from the outside.

Reply:
Right, okay, that works nicely.

Reply:
@wwwater what do you think about exposing
```hs
logRequestUUID :: MonadLog m => UUID -> m a -> m a
logRequestUUID = localData . logSendingResponse
```

Also: 
> `logSendingResponse` is badly named: it has nothing to do with sending a response!
> Maybe something like `requestUUID :: UUID -> [Pair]` is better?